package.json

```diff
@@ -51,7 +51,9 @@
     "url": "https://github.com/Chalarangelo/30-seconds-of-code/issues"
   },
   "homepage": "https://github.com/Chalarangelo/30-seconds-of-code#readme",
-  "dependencies": {},
+  "dependencies": {
+    "markdown-builder": "^0.8.3-dev"
+  },
```

Comment:
This is not a `dependecy`, it's a `devDependency` as you do not need the `markdown-builder` package to use the package.

Reply:
yep, you're absolutely right!